lib/IRGen/GenClangDecl.cpp

```diff
@@ -51,8 +59,11 @@ clang::Decl *getDeclWithExecutableCode(clang::Decl *decl) {
     if (initializingDecl) {
       return initializingDecl;
     }
+  } else if (auto rd = dyn_cast<clang::CXXRecordDecl>(decl)) {
```

Comment:
What about `clang::RecordDecl` instead? I assume we have this problem in C too? Or even better `clang::TagDecl` so we handle enums as well.

Reply:
This is outdated, we don't need it anymore.